a.env/Amir

```diff
@@ -0,0 +1 @@
+AMIR-12345
```

Comment:
firstly, it is wrong syntax of env variable defenition, it should be like `AMIR=123`
secondly, we don't need this variable, I told you that. 
thirdly, file name should be `.env`, not `a.env`